setup/mysql.sql

```diff
@@ -145,10 +147,25 @@ INSERT INTO `vexim`.`users`
     smtp, pop, realname, type, admin
 )
 VALUES 
-(   '1', 'siteadmin', 'siteadmin', '$1$12345678$2lQK5REWxaFyGz.p/dos3/', '65535', '65535', '', '',  'SiteAdmin', 'site', '1'
+(   '1', 'siteadmin', 'siteadmin', '$6$uR.EiB1dj5rrvwMF$Qh5LgdjOZavKXwhi9IF0Yuzu7qxsG.dLTTB8e./55ZRNfBuZVLnfUSOEXa0oWT6174myO.WYkOy83HYWAKPbK/', '65535', '65535', '', '',  'SiteAdmin', 'site', '1'
```

Comment:
Can't tell about all these products, never used them myself...

Regarding Jessie: indeed, if SHA512 is now used in /etc/shadow by default, it probably indicates that this is available in system `crypt()` now.

I don't quite understand your last point though. `$1$` is very likely to be uniformly supported, but even if not, the siteadmin user should not be logging into IMAP or POP3 anyway. For one, his home directories are not specified by default. I wonder if anyone ever tried logging in over IMAP as siteadmin. :)